src/main.rs

```diff
@@ -91,6 +96,9 @@ enum Commands {
     New {
         /// The project name.
         name: Option<String>,
+        /// New issue.
+        #[arg(short)]
+        issue: bool,
```

Comment:
can we add long command?

Reply:
Sure, I just forgot it, thanks for reminding me.